### PR TITLE
#164 updating the readme

### PR DESCRIPTION
**Pull request by sreevishnu211**
Changes are updating the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,12 +115,13 @@ GET `/`
 | Update an Address| PUT `/customers/{int:customer_id}/addresses/{int:address_id}`  
 | Delete an Address| DELETE `/customers/{int:customer_id}/addresses/{int:address_id}`           
 | List Addresses    | GET `/customers/{int:customer_id}/addresses`  
+| Search Customers and Addresses | GET `/customers/<query_field>=<query_value>`
```
Comment (Akshama) [inline README.md:118]: `/customers/?<query_field>=<query_value>`
Please put a question mark before the query field

File: README.md [hunk 2]
```diff
@@ -538,6 +546,45 @@ Failure Response : `HTTP_404_NOT_FOUND`
 }
 ```
 
+### Search Customers and Addresses
+
+URL : `http://127.0.0.1:8080/api/customers/<query_field>=<query_value>`
```
Comment (Akshama) [inline README.md:551]: Add the ? here as well before <query_field>

File: README.md (hunk 1 above)
Comment (sreevishnu211) [reply README.md:118]: oops sorry will do it

File: README.md (hunk 2 above)
Comment (sreevishnu211) [reply README.md]: done

**Pull request merged**